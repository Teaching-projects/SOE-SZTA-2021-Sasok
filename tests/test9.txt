111
21
123
32