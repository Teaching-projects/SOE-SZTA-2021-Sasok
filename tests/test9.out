Első egység HP-ja: Első egység DMG-je: Második egység HP-ja: Második egység DMG-je: 
A Csata elkezdődött! Warrior kezdheti a támadást Shaman ellen.
1. körben Warrior támad --- Warrior élete: 111 || Shaman élete: 102
2. körben Shaman támad, ---Warrior élete: 79 || Shaman élete: 102
2. körben Warrior támad --- Warrior élete: 79 || Shaman élete: 81
3. körben Shaman támad, ---Warrior élete: 47 || Shaman élete: 81
3. körben Warrior támad --- Warrior élete: 47 || Shaman élete: 60
4. körben Shaman támad, ---Warrior élete: 15 || Shaman élete: 60
4. körben Warrior támad --- Warrior élete: 15 || Shaman élete: 39
5. körben Shaman támad, ---Warrior élete: -17 || Shaman élete: 39

SHAMAN GYOZEDELMESKEDETT!
